cico_build_deploy.sh

```diff
@@ -15,6 +15,16 @@ mkdir -p dist && docker run --detach=true --name=almighty-ui-builder -e "API_URL
 # Build almigty-ui
 docker exec almighty-ui-builder npm install
 
+## Exec unit tests
+docker exec almighty-ui-builder ./run_unit_tests.sh
+
+if [ $? -eq 0 ]; then
```

Comment:
Instead of checking like this enable "exit on error" option by setting `set -e`. The `set -x` also can be added.

See this example for core: https://github.com/almighty/almighty-core/blob/master/cico_build_deploy.sh